src/components/item/item.md.scss

```diff
@@ -135,6 +148,15 @@ $item-md-sliding-content-background: $list-md-background-color !default;
     background-position: right ($item-md-padding-right - 2) center;
     background-size: 14px 14px;
   }
+  
+  [dir="rtl"] .item-md[detail-push] .item-inner,
+  [dir="rtl"] button.item-md:not([detail-none]) .item-inner,
+  [dir="rtl"] a.item-md:not([detail-none]) .item-inner {
+    @include svg-background-image($item-md-detail-push-svg-rtl);
+    padding-right: 0;
+    padding-left: 32px;
```

Comment:
I think in the theme file you need to have a variable:
'$x = 32px'
And then it should be in every file (iOS, md, wp) at the top, near all of the other variables, like:
'$x-ios = $x'